### ISSUE TITLE
some question about 𝐗_𝑡−𝜌𝐗_(𝑡−1) in the code

### ISSUE THREAD

**Issue by Lanturewen** [Daikon-Sun/AdjustAutocorrelation#4]
Hello!
 as the paper said，the input and output should be：
![image](https://user-images.githubusercontent.com/18157343/145926156-6ea992f0-b208-4916-ab4c-f78ef3a0c870.png)
but the code in forecasting_runner:
`  if args.inp_adj:
             x = torch.cat([avg[None].repeat(bs, 1, 1), x], dim=1)
             x = x[:, 1:] - rho * x[:, :-1]
    prd_y = self.model(x)
    if args.out_adj:
            prd_y += rho * x[:, -1]`
the x[:, -1] under has changed which mean that x[:, -1]=x[:, 1:] - rho * x[:, :-1] instead of x_{t-1}.
Do I misunderstand?

**Comment by Daikon-Sun** [comment]
Hi,
Yes, you are right.
Originally, this part is written in one line; but when I wanted to run the ablation study, I separated the input and output parts and thus caused the bug.
So, fortunately, the main experiments are run without the error, only the ablation study is affected by this bug.
Thanks for pointing the error out!
I will fix it.
